### PR TITLE
[improvement](profile)Add some block counter

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes

Add more counter to explain block_times counter.

### CODE REVIEW

**Review by yiguolei — DISMISSED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

File: be/src/pipeline/pipeline_task.cpp [hunk 1]
```diff
@@ -385,6 +388,11 @@ void PipelineTask::set_state(PipelineTaskState state) {
             COUNTER_UPDATE(_block_by_sink_counts, 1);
         } else if (state == PipelineTaskState::BLOCKED_FOR_RF) {
             _wait_bf_watcher.start();
+            COUNTER_UPDATE(_wait_bf_counts, 1);
+        } else if (state == PipelineTaskState::BLOCKED_FOR_DEPENDENCY) {
+            COUNTER_UPDATE(_wait_dependency_counts, 1);
```
Comment (yiguolei) [inline be/src/pipeline/pipeline_task.cpp:393]: ard. The VM will try to fix the stack guard now.
It's highly recommended that you fix the library with 'execstack -c <libfile>', or link it with '-z noexecstack'.
*** Query id: c409263e57a94900-8489ab59d02937e5 ***
*** Aborted at 1694784411 (unix time) try "date -d @1694784411" if you are using GNU date ***
*** Current BE git commitID: 4d65b1c ***
*** SIGSEGV unknown detail explain (@0x0) received by PID 8683 (TID 8988 OR 0x7f8dae03e700) from PID 0; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /root/doris/be/src/common/signal_handler.h:413
 1# os::Linux::chained_handler(int, siginfo*, void*) in /usr/local/software/jdk1.8.0_131/jre/lib/amd64/server/libjvm.so
 2# JVM_handle_linux_signal in /usr/local/software/jdk1.8.0_131/jre/lib/amd64/server/libjvm.so
 3# signalHandler(int, siginfo*, void*) in /usr/local/software/jdk1.8.0_131/jre/lib/amd64/server/libjvm.so
 4# 0x00007F8F7A8D1090 in /lib/x86_64-linux-gnu/libc.so.6
 5# doris::pipeline::PipelineTask::set_state(doris::pipeline::PipelineTaskState) at /root/doris/be/src/pipeline/pipeline_task.cpp:393
 6# doris::pipeline::PipelineXTask::execute(bool*) in /mnt/ssd01/pipline/OpenSourceDoris/clusterEnv/P0/Cluster0/be/lib/doris_be
 7# doris::pipeline::TaskScheduler::_do_work(unsigned long) at /root/doris/be/src/pipeline/task_scheduler.cpp:258
 8# doris::ThreadPool::dispatch_thread() at /root/doris/be/src/util/threadpool.cpp:541
 9# doris::Thread::supervise_thread(void*) at /root/doris/be/src/util/thread.cpp:466
10# start_thread at /build/glibc-SzIz7B/glibc-2.31/nptl/pthread_create.c:478
11# __clone at ../sysdeps/unix/sysv/linux/x86_64/clone.S:97

**Comment by wangbo** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by wangbo** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

File: be/src/pipeline/pipeline_task.cpp [hunk 2]
```diff
@@ -122,6 +122,9 @@ void PipelineTask::_init_profile() {
     _schedule_counts = ADD_COUNTER(_task_profile, "NumScheduleTimes", TUnit::UNIT);
     _yield_counts = ADD_COUNTER(_task_profile, "NumYieldTimes", TUnit::UNIT);
     _core_change_times = ADD_COUNTER(_task_profile, "CoreChangeTimes", TUnit::UNIT);
+    _wait_bf_counts = ADD_COUNTER(_task_profile, "WaitBfTimes", TUnit::UNIT);
```
Comment (zhiqiang-hhhh) [inline be/src/pipeline/pipeline_task.cpp:125]: What is the meaning of Bf

Comment (wangbo) [reply be/src/pipeline/pipeline_task.cpp:125]: bloomfilter

**Pull request merged**